src/sql-parser/src/parser.rs

```diff
@@ -2415,11 +2416,32 @@ impl<'a> Parser<'a> {
                 self.expect_keyword(TYPE)?;
                 KafkaSinkConfigOptionName::CompressionType
             }
-            PROGRESS => {
-                self.expect_keywords(&[GROUP, ID, PREFIX])?;
-                KafkaSinkConfigOptionName::ProgressGroupIdPrefix
-            }
-            TOPIC => KafkaSinkConfigOptionName::Topic,
+            PROGRESS => match self.expect_one_of_keywords(&[GROUP, TOPIC])? {
+                GROUP => {
+                    self.expect_keywords(&[ID, PREFIX])?;
+                    KafkaSinkConfigOptionName::ProgressGroupIdPrefix
+                }
+                TOPIC => {
+                    self.expect_keywords(&[TOPIC, REPLICATION, FACTOR])?;
```

Comment:
Looks like you have a double `TOPIC` here. Think you just want `[REPLICATION, FACTOR]`.

Reply:
Ah good catch, will update the sql parser tests so they would have caught this!